File: java-main/java-main/NestedforloopAssi/p10.java
/*write a program to print the following pattern
1 2 3 4
2 3 4 5
3 4 5 6
4 5 6 7*/

class Demo{

	public static void main(String[]args){
		
		int[][] arr = {{1, 2, 3, 4},
                       {2, 3, 4, 5},
                       {3, 4, 5, 6},
                       {4, 5, 6, 7}};
		for(int i = 0; i < 4; i++) {     /*for(int i=1;i<=4;i++){
		                                   int k=i;
		                                   for(int j=1;j<=4;j++){  
		                                   System.out.print(k++ + " ");
		                                   }
		                                   System.out.println();
		                                   }*/         
			for(int j = 0; j < 4; j++) {
               
				System.out.print(arr[i][j] + " ");
            
			}
		    	System.out.println();
		}
	}
}

File: flutter-main/flutter-main/A1/p10.dart
/*Write a dart program to calculate electricity bill of a house based
on following criteria
For first 90 units: No charge
90 to 180 units: 6 rupees per unit
180 to 250 units: 10 rupees per unit
Above 250 units : 15 rupees per unit
Input: 90
Output: 540
Input:120
Output:720*/

import "dart:io";

	void main() {
  	print("Enter the number of units:");
  	int units = int.parse(stdin.readLineSync()!);

  	double totalBill = 0.0;

  	if (units <= 90) {
    		totalBill = 0;
  	} else if (units <= 180) {
    		totalBill = (units - 90) * 6;
  	} else if (units <= 250) {
    		totalBill = (90 * 6) + (units - 180) * 10;
  	} else {
    		totalBill = (90 * 6) + (70 * 10) + (units - 250) * 15;
  	}

  		print("Total electricity bill: ${totalBill.toStringAsFixed(2)} rupees");
}

